driver/driver.go

```diff
@@ -213,7 +223,19 @@ func getExoscaleNodeMetadata() (*nodeMetadata, error) {
 		return nil, fmt.Errorf("node meta data Instance ID %s: %w", node.Spec.ProviderID, err)
 	}
 
-	zone, ok := v3.Zones[region]
+	var zonesToURL map[string]v3.URL
+
+	customZonesStr := os.Getenv("ZONE_TO_URL")
+	if customZonesStr != "" {
+		zonesToURL, err = customZones(customZonesStr)
+		if err != nil {
+			return nil, fmt.Errorf("couldn't read custom zone mapping %w", err)
+		}
+	} else {
+		zonesToURL = v3.Zones
+	}
+
+	zone, ok := zonesToURL[region]
```

Comment:
I'm thinking on the topic, because the zone mapping is not only used here in the CSI driver

Reply:
maybe it should go on egoscale then...